features/steps/project/source/browse_files.rb

```diff
@@ -19,6 +19,10 @@ class Spinach::Features::ProjectSourceBrowseFiles < Spinach::FeatureSteps
   step 'I see the ".gitignore"' do
     page.should have_content '.gitignore'
   end
+  
+  step 'I see the "user.feature"' do
+    page.should have_content 'user.feature'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.